lib/src/components/NxFileUpload/NxFileUpload.scss

```diff
@@ -47,6 +47,34 @@
   }
 }
 
+
+.nx-multi-file-upload {
+  width: 348px;
+
+  // need an enclosing div around NxMultiFileUpload that doesn't contain the validationError message
+  &__container {
+    $height: 220px;
+
+    min-height: $height;
```

Comment:
I think this should just be `height`.  I know the mockup says "minimum height" but as far as I can tell that's a designer-level note, given that there are no runtime conditions that would ever cause the component to _automatically_ exceed 220px tall.  I think the "minimum" phrasing in the mockup just means that downstream designers and devs are free to override the `height` we specify here with a different number as long as it's greater than 220px.

Actually, in the spirit of making that overriding easy, we should probably try to set the `height` on the top-level `.nx-multi-file-upload` class rather than making downstream devs dig into the component's interior classes.

Reply:
> I think this should just be `height`. 
> Actually, in the spirit of making that overriding easy, we should probably try to set the `height` on the top-level `.nx-multi-file-upload` class rather than making downstream devs dig into the component's interior classes.

Height added to `nx-multi-file-upload` here: [259bdb2](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/259bdb22c3ddc9356858bbec1ccefd6e66f7265c)